### DISCUSSION TITLE
Blazored.FluentValidation doesn't work on .Net Standard 2.1 library

### DISCUSSION THREAD

**Discussion by slavenl** [Blazored/FluentValidation#174] [Q&A] [answered]
Hi,

It seems that Blazored.FluentValidation v2.1.0 doesn't work on .Net standard 2.1 libraries.  Only .Net 6/7
Version 2.0.3 worked ok. 

Any comment, please?

**Comment by chrissainty** [comment, marked as answer]
You're correct. The new version doesn't target net standard anymore. The main reason for this is maintainability and being able to use the newest language features.

**Reply by slavenl** [reply]
Sad times for .net standard lovers :)
Ok, thank u for answer.
